tests/checkers/unittest_misc.py

```diff
@@ -116,3 +117,151 @@ def test_dont_trigger_on_todoist(self) -> None:
         """
         with self.assertNoMessages():
             self.checker.process_tokens(_tokenize_str(code))
+
+    @set_config(check_fixme_in_docstring=True)
+    def test_docstring_with_message(self) -> None:
```

Comment:
@Pierre-Sassoulas Do you want these unittests? I would be fine with having only the functional tests and remove these. They feel like duplicates

Reply:
Gotcha, that makes sense! Reverted changes to this file.